Dockerfile.ci

```diff
@@ -195,7 +195,7 @@ RUN mkdir -pv ${AIRFLOW_HOME} && \
     mkdir -pv ${AIRFLOW_HOME}/logs
 
 # Increase the value here to force reinstalling Apache Airflow pip dependencies
-ARG PIP_DEPENDENCIES_EPOCH_NUMBER="4"
+ARG PIP_DEPENDENCIES_EPOCH_NUMBER="5"
```

Comment:
I guess it depends on #13177 so it should be updated accordingly (or maybe merge those two PR).

Reply:
I merged these PRs.